### ISSUE TITLE
Responsive prefixes not working

### ISSUE THREAD

**Issue by Gyurmatag** [jrowlingson/stencil-tailwind#13]
Hi!

The package is great, but I noticed a problem with it:
Responsive prefixes are not working. [](https://tailwindcss.com/#responsive-to-the-core)
For example if I am using this:
`<div class="w-full sm:w-full md:w-1/3 lg:w-1/4 xl:w-1/6 m-4 text-white">Test</div>`
the responsiveness is not working. In my other project (Vue js) it's working nicely.

Can you please make a fix for this?

Thank you!

**Comment by newtonmunene99** [comment]
Hi @Gyurmatag , did you find a workaround to this?

**Comment by Gyurmatag** [comment]
No, I haven't found any. I think it's an issue, that need to be addressed from the developers of this library.

**Comment by nrawji** [comment]
I opened a PR to address this issue, it now correctly includes responsive classes.

**Comment by newtonmunene99** [comment]
> I opened a PR to address this issue, it now correctly includes responsive classes.

Amazing, for now I decided to use tailwindcss as a postcss plugin with the downside that I won't be able to use shadow dom. I'll give a try, thanks!
